src/v3/src/components/Form/Form.tsx

```diff
@@ -30,10 +30,15 @@ const Form: FunctionComponent<{
     idxTransaction: currTransaction,
     setMessage,
     dataSchemaRef,
+    setWidgetRendered,
   } = useWidgetContext();
   const onSubmitHandler = useOnSubmit();
   const onValidationHandler = useOnSubmitValidation();
 
+  useEffect(() => {
+    setWidgetRendered(true);
+  });
```

Comment:
Should we consider resetting this flag after each re-render? Or in `useOnSubmit` hook when we set the new transaction? (No need to consider that now for this PR but just thinking about the future.

Reply:
Good call, I don't think this works when you move to a second view right now. Let me investigate to see if we can make this better

Reply:
@glenfannin-okta Good callout! For parity, the `afterRender` _should_ get called after each screen is painted.